scripts/deploy-infra.sh

```
@@ -466,6 +476,59 @@ deploy_codebuild_stack() {
+    exit 1
+  }
+
+  # Push to ECR
+  info "Pushing image to ECR..."
+  docker push "$repo_uri:latest" >/dev/null || {
+    err "Failed to push Docker image to ECR"
+    exit 1
+  }
+
+  info "Successfully pushed Docker image to $repo_uri:latest"
+  POST_LAMBDA_IMAGE_PUSH_PENDING=0
```

Shellcheck (0.11.0)

[warning] POST_LAMBDA_IMAGE_PUSH_PENDING appears unused. Verify use (or export if used externally).

(SC2034)